docs/v3.x/guides/custom-admin.md

```diff
@@ -87,7 +87,7 @@ Let's eject the file to be able to customize it.
 
 In this new file, paste the current [dateFormats](https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-content-manager/admin/src/utils/dateFormats.js) code.
 
-To run your application, you will have to run the `yarn develop --watch-admin` command.
+To run your application, you will have to run the `yarn develop --watch-admin` / `npm run develop -- --watch-admin` command.
```

Comment:
Same tabs?

Reply:
Yes, same thing here, tabs should be used!